component.yml.tpl

```diff
@@ -29,6 +29,7 @@ phases:
             - eval "$(ssh-agent -s)"
             - ssh-add ~/.ssh/git_rsa
             %{~ endif ~}
+            - rm -rf ansible-repo
```

Comment:
is it caching it somehow?

Reply:
I ran into the issue building on top of a previously built image, which probably indicates an issue with the teardown module or how I'm using it? But this seems like a.. harmless sanity check addition here that was a way quicker solution